apps/src/sites/studio/pages/home/_link_blocks.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import GradientNavCard from '@cdo/apps/templates/teacherHomepage/GradientNavCard';
+
+$(document).ready(showLinkBlocks);
+
+function showLinkBlocks(linkBlocks) {
+  const linkBlocksData = document.querySelector('script[data-linkblocks]');
+  const config = JSON.parse(linkBlocksData.dataset.linkblocks);
+
+  ReactDOM.render(
+    <div>
+      {config.map((item, index) => (
+        <GradientNavCard
+          key={index}
+          title={item.text}
+          link={item.link}
+          description=""
+          image="http://code.org/img.png"
+          buttonText="Learn more"
+        />
+      ))}
+    </div>,
```

Comment:
This was done because we don't currently have a single root react DOM for the whole /courses page, but that should change shortly in this PR, at which point that will be possible.